duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/inputscreen/ui/InputScreenFragment.kt

```diff
@@ -248,6 +271,14 @@ class InputScreenFragment : DuckDuckGoFragment(R.layout.fragment_input_screen) {
         onInputFieldClicked = {
             viewModel.onInputFieldTouched()
         }
+        onTabTapped = { index ->
+            if (binding.viewPager.currentItem != index) {
+                viewModel.onTabTapped()
+                if (!viewModel.hasNewTabContent()) {
+                    animateLogoToPosition(index)
+                }
+            }
+        }
```

Comment:
Could this logic be moved to the ViewModel where `animateLogoToPosition` is invoked via a command?

```
viewModel.onTabTapped(isSearchTab = index == 0)
```
or something along these lines

Reply:
Moved to ViewModel